src/main/java/frc/robot/subsystems/instances/Bling.java

```diff
@@ -91,8 +95,45 @@ public void setLEDs2(int i, int i2, int r, int g, int b) {
     m_ledBuffer.setRGB(i2, r, g, b);
     m_led.setData(m_ledBuffer);
   }
+  
+  public void setLEDFromColor(int i) {
+    Color blue = Color.kBlue;
+    m_ledBuffer.setLED(i, blue);
+  }
 
   public AddressableLEDBuffer getM_LEDBuffer() {
     return m_ledBuffer;
   }
+
+  public void LEDRainbow() {
+    int numLEDs = m_ledBuffer.getLength();
+    for (int i = 0; i < numLEDs; i++) {
+      if (((i + move_rainbow) % 12) == 0 || ((i + move_rainbow) % 12) == 1) {
```

Comment:
I still think the switch works here - just needs to be reworked a little

Reply:
Has it been tested recently?

Reply:
sorry, I'm proposing she try another way to code this
as written, the code works